exchanges/localbitcoins/localbitcoins.go

```diff
@@ -768,7 +764,13 @@ func (l *LocalBitcoins) SendAuthenticatedHTTPRequest(ep exchange.URL, method, pa
 	headers["Content-Type"] = "application/x-www-form-urlencoded"
 
 	if l.Verbose {
-		log.Debugf(log.ExchangeSys, "Sending POST request to `%s`, path: `%s`, params: `%s`.", endpoint, path, encoded)
+		log.Debugf(log.ExchangeSys, "%s Sending `%s` request to `%s`, path: `%s`, params: `%s`.",
```

Comment:
[DEBUG]|EXCHANGE| 26/02/2021 10:02:26 |LocalBitcoins Sending `GET` request to `https://localbitcoins.com`, path: `/api/wallet-balance/`, params: ``.

👍